runners/katacoda/index.ts

```diff
@@ -110,6 +118,25 @@ export class Katacoda extends Runner {
         });
         this.renderTemplate("cobiGenJava.md", this.outputPathTutorial + "step" + (this.stepsCount++) + ".md", { text: step.text, textAfter: step.textAfter, javaFile: params[0], cobiGenTemplates: cobiGenTemplates });
         return null;
+
+    }
+
+    runCreateDevon4jProject(step: Step, command:Command): RunResult {
+
+        // generate template to change directory, if the current directory is not equal to the required start directory
+       let cdCommand = this.changeCurrentDir(path.join("/root", "devonfw"));
+       let cdTarget = this.changeCurrentDir(path.join(command.parameters[1]));
```

Comment:
Is it necessary to pass this parameter. In the task for the console runner we pass only the project name (https://github.com/devonfw-forge/tutorial-compiler/commit/a31e110b6652d106552cdd5e9a817100a6b79a28). I think you can assume that the project will always be generated in the workspace/main folder.